x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -285,9 +285,7 @@ export const ConfigurationStepForm: FC<ConfigurationStepProps> = ({
 
       const formStateUpdated = {
         ...(shouldUpdateModelMemoryLimit ? { modelMemoryLimit: expectedMemory } : {}),
-        ...(depVarIsRuntimeField || jobTypeChanged || depVarNotIncluded
-          ? { includes: formToUse.includes }
-          : {}),
```

Comment:
`jobTypeChanged` was removed here because it caused extra renders of the component that prevented the `includedFields` table on the Regression job page not to be automatically populated when `dependentVariable` was selected. This occurred because this condition was detecting a job change when there wasn't one.